test/hotspot/jtreg/compiler/splitif/TestSplitDivisionThroughPhi.java

```diff
@@ -78,6 +102,27 @@ static void testPushDivIThruPhi() {
         }
     }
 
+    // Fixed with JDK-8336792.
```

Comment:
Do you want to add this bug-number to the `@bug` above?

Reply:
I've already added it for the newly added runs but I guess it does not hurt to add it to the existing ones as well.